src/appengine/handlers/cron/triage.py

```diff
@@ -217,6 +231,11 @@ def is_similar_bug_open_or_recently_closed(testcase, issue_tracker):
         issue.closed_time,
         compare_to=datetime.datetime.utcnow(),
         hours=data_types.MIN_ELAPSED_TIME_SINCE_FIXED)):
+      _add_triage_message(
+          testcase,
+          ('Bug filing is on temporary hold since similar testcase '
```

Comment:
Nit: "Delaying filing a bug" instead of "Bug filing is on temporary hold".

Reply:
Done!